test/TestUtils.py

```diff
@@ -185,6 +194,12 @@ def test_get_yaml_files_git_verbose(
     ids=('in Git', 'outside Git'),
 )
 def test_get_yaml_files_silent(is_in_git, monkeypatch, capsys):
```

Comment:
You should also check it with `caplog` because originally this test was added to check that there's no warnings when falling back to pure-python lookup.

Reply:
I am inclined to believe that using capsys is more comprehensive than caplog, mainly because what caplog captures is a subset of what capsys does, usually they are the same but there is no guarantee that someone does not slip a `print()` by mistake.

Anyway, I do not have any strong feelings about the test so I would likely accept any edit made on it.